homeassistant/components/panasonic_viera/entity.py

```diff
@@ -0,0 +1,26 @@
+"""Base entity for Panasonic Viera."""
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.trigger import PluggableAction
```

Comment:
The name of this class is not good IMO, we do have something called device actions, yet we have a trigger helper named `PluggableAction` with this meaningless docstring `"""A pluggable action handler."""`.

Can this class please be renamed and/or its documentation improved?